pytest_watcher/watcher.py

```diff
@@ -15,12 +17,17 @@
 trigger = None
 
 
+logger = logging.getLogger(__name__)
+
+
 @dataclass
 class ParsedArguments:
     path: Path
     now: bool
     delay: float
     runner: str
+    include_filter: str
```

Comment:
I suggest we use `include_pattern` and `exclude_pattern` as names for these arguments to make things a bit clearer